src/engine/cql-to-elm.ts

```diff
@@ -1,9 +1,74 @@
 // External CQL -> ELM service
 import config from 'utils/ConfigManager';
+import {
+  extractJSONContent,
+  extractMultipartBoundary,
+  extractMultipartFileName
+} from 'utils/regexes';
 import { ElmLibrary } from 'elm-model';
 
 const url = config.get('cqlToElmWebserviceUrl');
 
+export interface CqlObject {
+  main: string;
+  libraries: Library;
+}
+
+export interface Library {
+  [name: string]: string; // should probably have an object for expected ELM structure.
```

Comment:
Is this interface for a CQL Library?

Reply:
This came from Camino, I forget offhand if I added this as part of my current WIP PR or if it was already there. Just a wrapper for a mapping of `name of CQL library` -> `text of cql library`, it's not a cql or elm library by itself

Reply:
Ahh yes it's used by `CqlObject`, I brought this over from camino. Should I re-name this to `CqlLibrary`?

Reply:
I think that would be helpful. Is the comment still relevant?

Reply:
I removed the comment and the interface since it's not being used elsewhere and just explicitly declared in the `CqlObject` interface.